### ISSUE TITLE
Batch Normalization axis

### ISSUE THREAD

**Issue by jjjkkkjjj** [nyck33/TrackNetMirror#1]
Hi, when I see your code for implementation, i have a question about batch normalization.

Here is your building model's code;
```python
#layer1
x = Conv2D(64, (3, 3), kernel_initializer='random_uniform', padding='same', data_format='channels_first' )(imgs_input)
x = ( Activation('relu'))(x)
x = ( BatchNormalization())(x)

#layer2
x = Conv2D(64, (3, 3), kernel_initializer='random_uniform', padding='same', data_format='channels_first' )(x)
x = ( Activation('relu'))(x)
x = ( BatchNormalization())(x)

...


```

I think you should set `BatchNormalization(axis=1)` if the input tensor has **channels_first** order.
Default is for **channels_last**

**Comment by nyck33** [comment]
@jjjkkkjjj it's been super long since I used this and I remember there was an issue with this when I was trying to add more frames.  However, this is a cloned repo from Gitlabs.  Not my code actually.

**Comment by jjjkkkjjj** [comment]
I got it.
I thought you are TrackNet's author.
